docs/content/ingestion/stream-pull.md

```diff
@@ -99,6 +99,9 @@ The property `druid.realtime.specFile` has the path of a file (absolute or relat
       "basePersistDirectory": "\/tmp\/realtime\/basePersist",
       "rejectionPolicy": {
         "type": "serverTime"
+      },
+      "indexSpec": {
+         "bitmap":"roaring"
```

Comment:
please don't include non-default tunings in examples, this just causes confusion.